scripts/version/Version.props

```diff
@@ -1,6 +1,6 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MainVersion>8.1.0</MainVersion>

```

Comment:
Hello y'all ( @costin-zaharia-sonarsource , @gregory-paidis-sonarsource ), does this mean that you'll release a new major version (9.0.0) soon? I was under the impression initially that it should have been a new patch release to address comments on the community, but maybe I've misunderstood!

I'm asking to know whether we'll need another major version bump on AzDO

Reply:
We are not yet 100% sure how we'll proceed next. Since the scope of the release increased, we are considering first releasing a 9.0.0 release candidate to let the users test the latest changes before the actual release.

Reply:
Okay thanks for the heads up @costin-zaharia-sonarsource !